decidim-proposals/db/migrate/20171220084719_add_published_at_to_proposals.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class AddPublishedAtToProposals < ActiveRecord::Migration[5.1]
+  def up
+    add_column :decidim_proposals_proposals, :published_at, :datetime, index: true
+    Decidim::Proposals::Proposal.find_each do |proposal|
```

Comment:
This will take a lot of time in some installations, can you change it to the following code please?

`Decidim::Proposals::Proposal.update_all("published_at = updated_at")`

Reply:
I did it like you mention, but didn't pass CodeClimate checks:

https://github.com/decidim/decidim/commit/976934aacf7481f4136dfc678375b149270521bd#diff-60f82c2463333244ee2dfc680da8548c

Reply:
I'd add a Rubocop exception there then, since the performance hit could be quite hard:

```ruby
# rubocop:disable Rails/SkipsModelValidations
Decidim::Proposals::Proposal.update_all("published_at = updated_at")
# rubocop:enable Rails/SkipsModelValidations
```

Reply:
Ok, I've change it